src/facade/borrow/borrow.ts

```diff
@@ -126,13 +121,9 @@ export class Borrow {
           const userBalance = await queryCustodyBorrower({
             lcd: this._lcd,
             ...getCollateralsOption,
-            custody: getCollateralsOption.market,
+            custody: whitelist.custody_contract,
           })(this._addressProvider);
 
-          if (userBalance.balance === '0') {
-            return null;
-          }
```

Comment:
There was a balance check here which returns null. It's a bit weird since if this condition is met you get something like the following back from this function if your balance is 0:

```
getCollateralValue({ market: 'uusd', address: '...'  }) => [ null ]
```

Better to just return an actual entry e.g.

```
[
  {
    collateral: 'terra1....',
    balance: '0.000000000000000000'
  },
  {
    collateral: 'terra1....',
    balance: '0.000000000000000000'
  }
]
```

Reply:
Just my opinion but this was added to make these functions work properly again